.moban.yaml

```diff
@@ -18,7 +18,7 @@ packages:
 dependencies:
   - git+https://gitlab.com/coala/coala-utils.git
   - git-url-parse
-  - django
+  - django==2.1.7
```

Comment:
The "problem" is django 2.2 , so it should be `django>2.1,<2.2`

It would also be acceptable to not touch this at all, allowing any version of Django including 2.2, and use xenial image in our CI.

The problem is our CI, not Django.

Reply:
Yes! I also figures out first that the problem was due to different ubuntu version ie. trusty which was having sqlite v3.8.2 installed and doesn’t support later version of sqlite which was a prerequisite of django 2.2.1 which was getting installed from `requirements.txt`. Then later on i checked that django v2.2.1 was getting installed which i wasn’t familiar with and tried to roll back to v2.1.7 at which our builds were green. 
So, we can change to ubuntu xenial image by specifying `dist: xenial` in travis.yaml, but on netlify when I’ve changes my deployed fork repo to run on ubuntu xenial it wasn’t getting executed. The build was getting failed due to an unkonown error. Can you please check into it by changing settings for coala-communtiy repo on netlify. If the build gets failed then I guess we have to stick with ubuntu trusty version with django v2.1.x